src/main/java/mdlaf/components/titlepane/MaterialTitlePaneUI.java

```diff
@@ -520,8 +522,8 @@ protected void paintComponent(Graphics g) {
                 theTitle = SwingUtilities2.clipStringIfNecessary(rootPane, fm, theTitle, titleW);
                 xOffset -= SwingUtilities2.stringWidth(rootPane, fm, theTitle);
             }
-            int titleLength = SwingUtilities2.stringWidth(rootPane, fm, theTitle);
-            SwingUtilities2.drawString(rootPane, g, theTitle, xOffset, yOffset);
+            int titleLength = (int)BasicGraphicsUtils.getStringWidth(rootPane, fm, theTitle);
```

Comment:
Please remove restore this change

Reply:
I restored to the StringUtilities2 function. In the longer term, we need to remove this - it is not a public API and will likely break in future Java versions

Reply:
I don't want to accelerate the times, when java will decide to remove this API, should insert a method to support the previous versions of JDK or We will scheduler something, but when after the Oracle remove this API not now.

Swing is an old framework and I want to support also the old versions, at the moment I can compile with the JDK13 the library with StringUtilities2.

Thanks for comprehension.